e2e/playwright/tests/folder-rules/page-objects/components/actions-dropdown.component.ts

```diff
@@ -10,23 +10,84 @@ import { Locator, Page } from '@playwright/test';
 import { BaseComponent } from '@alfresco/playwright-shared';
 
 export enum ActionType {
-  Aspect = 'Add aspect',
+  AddAspect = 'Add aspect',
   SimpleWorkflow = 'Add simple workflow',
-  IncrementCounter = 'Increment Counter'
+  IncrementCounter = 'Increment Counter',
+  CheckIn = 'Check in',
+  AutoDeclareOptions = 'Auto-Declare Options',
+  CheckOut = 'Check out',
+  Copy = 'Copy',
+  ExecuteScript = 'Execute script',
+  ExtractCommonMetadataFields = 'Extract common metadata fields',
+  FileAsRecord = 'File as Record',
+  FileVersionAsRecord = 'File Version as Record',
+  HideRecord = 'Hide Record',
+  Import = 'Import',
+  Move = 'Move',
+  RemoveAspect = 'Remove aspect',
+  RequestAIRenditions = 'Request AI renditions',
+  SendEmail = 'Send email',
+  SetPropertyValue = 'Set property value',
+  SpecialiseType = 'Specialise type',
+  TransformAndCopyContent = 'Transform and copy content',
+  TransformAndCopyImage = 'Transform and copy image'
 }
 
 export class ActionsDropdownComponent extends BaseComponent {
-  private static rootElement = '.mat-select-panel';
+  private static rootElement = 'aca-rule-action-list';
 
-  public getOptionLocator = (optionName: string): Locator => this.getChild('.mat-option-text', { hasText: optionName });
+  public getOptionLocator = (optionName: string): Locator => this.page.locator('.mat-select-panel .mat-option-text', { hasText: optionName });
+  private ruleActionLocator = this.getChild('aca-rule-action [data-automation-id="rule-action-card-view"]');
+  private addActionButtonLocator = this.getChild('[data-automation-id="rule-action-list-add-action-button"]');
+  private actionDropdownLocator = this.getChild('[data-automation-id="rule-action-select"]');
+  private actionAspectNameLocatorText = '[data-automation-id="header-aspect-name"] mat-select';
```

Comment:
the suffixes `Text` are confusing, as you select HTML elements, not the text, I suggest dropping the Text from variables